src/compiler/transformers/declarations.ts

```diff
@@ -1471,7 +1474,7 @@ export function transformDeclarations(context: TransformationContext) {
                             return undefined; // unique symbol or non-identifier name - omit, since there's no syntax that can preserve it
                         }
                         getSymbolAccessibilityDiagnostic = createGetSymbolAccessibilityDiagnosticForNode(p.valueDeclaration);
-                        const type = resolver.createTypeOfDeclaration(p.valueDeclaration, fakespace, declarationEmitNodeBuilderFlags | NodeBuilderFlags.NoSyntacticPrinter, symbolTracker);
+                        const type = resolver.createTypeOfDeclaration(p.valueDeclaration, fakespace, declarationEmitNodeBuilderOptions.flags | NodeBuilderFlags.NoSyntacticPrinter, symbolTracker);

```

Comment:
This is the sort of thing where this approach breaks without some helpers; this call site, for example, loses the other node builder options because it pulls off just the flags.

Reply:
Yeah, that's definitely a bug I have missed. This could be easily fixed by doing `{ ...declarationEmitNodeBuilderOptions, flags: declarationEmitNodeBuilderOptions.flags | NodeBuilderFlags.NoSyntacticPrinter }`